extensions/positron-zed/src/positronZedLanguageRuntime.ts

```diff
@@ -381,16 +382,17 @@ export class PositronZedRuntimeSession implements positron.LanguageRuntimeSessio
 				this.shutdown();
 			}
 			return;
-			// } else if (match = code.match(/^view( .+)?/)) {
-			// 	// Simulate a data viewer
-			// 	const title = (match.length > 1 && match[1]) ? match[1].trim() : 'Data';
-			// 	this.simulateDataView(id, code, `Zed: ${title}`);
-			// 	return;
+		} else if (match = code.match(/^exec silent ([a-zA-Z]+) (.+)/)) {
+			// Execute code silently in another language.
+			const languageId = match[1];
+			const codeToExecute = match[2];
+			this.simulateCodeExecution(id, code, languageId, codeToExecute, positron.RuntimeCodeExecutionMode.Silent);
+			return;
 		} else if (match = code.match(/^exec ([a-zA-Z]+) (.+)/)) {
 			// Execute code in another language.
 			const languageId = match[1];
 			const codeToExecute = match[2];
-			this.simulateCodeExecution(id, code, languageId, codeToExecute);
+			this.simulateCodeExecution(id, code, languageId, codeToExecute, positron.RuntimeCodeExecutionMode.Interactive);
```

Comment:
The default value that code is executed as is `RuntimeCodeExecutionMode.Interactive` if a `RuntimeCodeExecutionMode` isn't provided so this is change is a 50/50 on if we want to explicitly provide the value here or not.